### PR TITLE
fix(build): windows executes build correctly

### PR DESCRIPTION
**Pull request by Yueyanc**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Review by sxzz — COMMENTED** [review]
I think we should refactor this part. And using `fs-extra` to copy and remove file

**Comment by ChenAdien** [comment]
> 我认为我们应该重构这部分。并`fs-extra`用于复制和删除文件
Ok, I'll try to reconstruct it myself

**Comment by ChenAdien** [comment]
> I think we should refactor this part. And using `fs-extra` to copy and remove file

Ok, I'll try to reconstruct it myself

**Comment by JeremyWuuuuu** [comment]
So umm, I myself is not very familiar with Windows Platform APIs for `rsync` etc, but for me personally I think for Windows users, WSL2 would be a better solution without all those API mappings.

Just in case we should add some documentation here for recording the mapped APIs such as `rsync`.

File: build/utils/process.ts [hunk 1]
```diff
@@ -4,7 +4,37 @@ import { projRoot } from './paths'
 
 export const run = async (command: string, cwd: string = projRoot) =>
   new Promise<void>((resolve, reject) => {
-    const [cmd, ...args] = command.split(' ')
+    let [cmd, ...args] = command.split(' ')
+    args.forEach((item, index) => {
+      console.log(item)
```
Comment (JeremyWuuuuu) [inline build/utils/process.ts:9]: Also please don't forget to remove the console part.

**Pull request closed without merging**